packages/functional-tests/tests/react-conversion/oauthSignin.spec.ts

```diff
@@ -8,25 +8,14 @@ const AGE_21 = '21';
 
 test.describe('severity-1 #smoke', () => {
   test.describe('react OAuth signin', () => {
-    test.beforeEach(async ({ pages: { configPage } }) => {
-      const config = await configPage.getConfig();
-      test.skip(
```

Comment:
React versions of Sign in and sign up flows have been fully rolled out in production for some time, no longer need to support backbone